src/evaluation/term_methods.h

```diff
@@ -106,8 +112,27 @@ constexpr static inline TermScore getKnightScore(const BitBoard& board, const ui
 
         if constexpr (player == PlayerWhite) {
             ADD_SCORE_INDEXED(psqtKnights, pos);
+
+            uint64_t square = helper::positionToSquare(pos);
```

Comment:
square is used for both white and black so can be generalized (outside of player check)